File: allure-junit5/src/test/java/io/qameta/allure/examples/junit5/AllureFixtureTest.java
package io.qameta.allure.examples.junit5;

import org.junit.jupiter.api.*;

import static io.qameta.allure.Allure.step;

public class AllureFixtureTest {

    @BeforeAll
    public static void beforeAll() {
        step("Step inside beforeAll");
    }

    @BeforeEach
    public void beforeEach() {
        step("Step inside beforeEach");
    }

    @Test
    public void allureFixtureTest() {
        step("Step inside allureFixtureTest");
    }

    @AfterEach
    public void afterEach() {
        step("Step inside afterEach");
    }

    @AfterAll
    public static void afterAll() {
        step("Step inside afterAll");
    }

}

File: kotlin-junit5-gradle-kts/src/test/kotlin/io/qameta/allure/examples/junit5/AllureFixtureTest.kt
package io.qameta.allure.examples.junit5

import io.qameta.allure.Allure.step
import org.junit.jupiter.api.*

class AllureFixtureTest {

    companion object {
        @JvmStatic
        @BeforeAll
        fun beforeAll() {
            step("Step inside beforeAll")
        }

        @JvmStatic
        @AfterAll
        fun afterAll() {
            step("Step inside afterAll")
        }
    }

    @BeforeEach
    fun beforeEach() {
        step("Step inside beforeEach")
    }

    @Test
    fun allureFixtureTest() {
        step("Step inside allureFixtureTest")
    }

    @Test
    fun allureFixtureTest2() {
        step("Step inside allureFixtureTest2")
    }

    @AfterEach
    fun afterEach() {
        step("Step inside afterEach")
    }

}